bazel/core.bzl

```diff
@@ -1,10 +1,97 @@
-# MLN_RENDERER_DEFINES = select({
-#     ":legacy_renderer": ["MLN_LEGACY_RENDERER=1"],
-#     "//conditions:default": ["MLN_LEGACY_RENDERER=0"]
-# }) + select({
-#     ":drawable_renderer": ["MLN_DRAWABLE_RENDERER=1"],
-#     "//conditions:default": ["MLN_DRAWABLE_RENDERER=0"]
-# })
+MLN_PUBLIC_GENERATED_STYLE_HEADERS = [
```

Comment:
I am still a bit worried about how we can best keep this file up to date.

The generation scripts output a list of files, maybe we can write those out to a `.bzl` file?

Reply:
It does, I'm looking at how to control where that gets placed as it doesn't look to be explicitly created and is rather an artifact of the `ejs` module being used.

For context, the [CMake](https://github.com/maplibre/maplibre-native/blob/9374765cb873306606cbd2679958e8d1e2fe9a0f/CMakeLists.txt#L133) had everything mixed together in opaque bricks.

Reply:
> For context, the [CMake](https://github.com/maplibre/maplibre-native/blob/9374765cb873306606cbd2679958e8d1e2fe9a0f/CMakeLists.txt#L133) had everything mixed together in opaque bricks.

:face_in_clouds: